changelog.md

```diff
@@ -17,6 +17,7 @@
 
 ### Fixes
 
+* [45](https://github.com/zeta-chain/security/issues/45) - Outbound tx confirmation/inclusion enhancement
```

Comment:
This repo is not publicly available, we can link the PR instead

Reply:
[commit](https://github.com/zeta-chain/node/pull/1530/commits/4d24ba320d56bdbddb1bb666246495503df896af)